src/components/header/components/controlOptionsList/ControlOptionsList.js

```diff
@@ -0,0 +1,38 @@
+// Libraries
+import PropTypes from 'prop-types'
+import _map from 'lodash/map'
+
+// component
+import ControlOption from "../controlOption";
+
+function getControlOptionJsx(item, index){
+    return <ControlOption key={item.label+index} {...item} />
+}
+
+function ControlOptionsList(props){
+
+    const { controlOptionsData } = props;
+
+    const list = _map(controlOptionsData, getControlOptionJsx)
+
+    return (
+        <> {list} </>
+    );
+}
+
+ControlOptionsList.propTypes = {
+    controlOptionsData : PropTypes.arrayOf(PropTypes.object)
```

Comment:
the object has a defined shape, so you can use PropTypes.arrayOf(PropTypes.shape({.  .....  }))